src/views/SpecialResolution.vue

```diff
@@ -37,7 +37,7 @@
         <AlterationSummary
           class="mt-10"
           :validate="getAppValidate"
-          @haveChanges="onAlterationSummaryChanges()"
+          @haveChanges="onSpecialResolutionSummaryChanges()"
```

Comment:
You're using the AlterationSummary component. Does it work for you or do you need your own? If you're reusing it, please look into renaming it (and moving its location from alteration-only to common or something).